index.bs

```diff
@@ -1234,95 +1246,103 @@ A raw cryptographic signature must assert the integrity of both the client data
-what attestation types are acceptable to them by policy. [RPS] will also need to understand the characteristics of the
-authenticators that they trust, based on information they have about these authenticators. For example, the FIDO Metadata
-Service [[FIDOMetadataService]] provides one way to access such information.
+It is expected that most authenticators will support a small number of attestation types and attestation statement formats,
+while [RPS] will decide what attestation types are acceptable to them by policy. [RPS] will also need to understand the
+characteristics of the authenticators that they trust, based on information they have about these authenticators. For example,
+the FIDO Metadata Service [[FIDOMetadataService]] provides one way to access such information.
 
 
-### Attestation Formats ### {#attestation-formats}
+### Attestation Statement Formats ### {#attestation-formats}
 
-As described above, an attestation format is a data format which represents a cryptographic signature by an authenticator over a
-set of contextual bindings. Each attestation format is defined by the following attributes:
+As described above, an attestation statement format is a data format which represents a cryptographic signature by an
+authenticator over a set of contextual bindings. Each attestation statement format is defined by the following attributes:
 
-- The name of the format, used to identify it in a {{AuthenticationAttestation}} structure. This MUST be an ASCII string, and MUST
-    NOT be an <a>ASCII case-insensitive match</a> for the name of any other attestation format.
+- Its <a>attestation format identifier</a>.
 
 - The set of attestation types supported by the format.
 
 - The syntax of an attestation statement produced in this format.
 
-- The procedure for computing an attestation statement in this format given the <a>attToBeSigned</a> for the attestation,
-    created as per [[#generating-an-attestation-statement]].
-
-- The procedure for verifying an attestation statement, takes the following inputs:
-    - The <a>authenticator data</a> claimed to have been used for the attestation,
-    - The <a>clientDataHash</a> of the client's contextual bindings,
-    - A trust anchor (a root certificate, a DAA root key, or the <a>credential public key</a> itself),
+- The procedure for computing an attestation statement in this format given the credential to be attested, the
+    <a>authenticatorData</a> for the attestation, and a <a>clientDataHash</a>.
 
-    and returns a Boolean value indicating whether the attestation is cryptographically valid, and if so the attestation type.
+- The procedure for verifying an attestation statement, which takes as inputs the <a>authenticatorData</a> claimed to have been
+    used for the attestation and the <a>clientDataHash</a> of the client's contextual bindings, and returns either:
```

Comment:
Reading this again, I think we can maybe reduce text and simplify. Now that we don't have that much in common between the attestation and assertion signatures, we can just delete the signature format section and absorb the signing and verification subsections into the attestation and assertion sections. Will try this out in the next commit.